hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -373,9 +373,9 @@
     }
     {
       name: chip_sw_sleep_pin_mio_dio_val
-      desc: '''Verify the MIO output values in deep sleep state.
+      desc: '''Verify the MIO output values in any sleep states.
 
-            SW programs the MIO OUTSEL CSRs to to ensure that in deep sleep it randomly picks
+            SW programs the MIO OUTSEL CSRs to to ensure that in sleep it randomly picks
```

Comment:
nit: `to to` -> `to`

As for the `random` part in `chip_sw_random_sleep_pin_mio_dio_val` that we are attempting to merge into this test point, I think the idea was to randomly select the sleep mode and do multiple sleep entry / exist iterations (in addition to just randomly picking the tie values)... @tjaychen @sriyerg do you remember?

Reply:
@eunchan I think you mentioned that normal / deep sleep is chosen randomly, right? 
can you add that to the description as well, just to be clear?